languages/go/.gitignore

```diff
@@ -1 +1,2 @@
-oso
+internal/ffi/native/**/*.[ah]
+internal/ffi/native/**/*.lib
```

Comment:
The new `languages/go/internal/ffi/native/macos/` subdirectories (`a[rm]d64`) weren't captured by the previous `.gitignore` patterns